src/Shared/E2ETesting/selenium-config.json

```diff
@@ -1,7 +1,7 @@
 {
   "drivers": {
     "chrome": {
-      "version" : "99.0.4844.35"
+      "version" : "106.0.5249.21"
```

Comment:
@MackinnonBuck this is a bigger change than in `main` and 7.0, is there anything else we should update?

Reply:
We might be able to undo the change to this specific file for this PR - I don't think it was crucial for the actual pipeline fix. But if we did take this change, we would probably want to update the Selenium versions in `Versions.props` as well (probably to match what they are in `main`/`release/7.0`.

Reply:
> We might be able to undo the change to this specific file for this PR - I don't think it was crucial for the actual pipeline fix.

I feel like keeping this updated, across releases is important. If there's a browser regression or something else, we want to know it impacts .NET 6, rather than just knowing it worked with .NET 6 with the driver at the time of .NET 6's release.